presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableHandle.java

```diff
@@ -33,19 +33,22 @@ public class IcebergTableHandle
     private final TableType tableType;
     private final Optional<Long> snapshotId;
     private final TupleDomain<IcebergColumnHandle> predicate;
+    private final boolean specifiedSnapshot;
```

Comment:
Not sure whether this is a good name for a boolean variable. `specifiedSnapshot` sounds like a noun. Do you mean `snapshotSpecificed`?

Reply:
Indeed, yes. I will fix it.